single-line-diagram-core/src/test/resources/VoltageIndicatorLibrary/components.css

```diff
@@ -0,0 +1,7 @@
+/* ----------------------------------------------------------------------- */
+/* File : components.css ------------------------------------------------- */
+.sld-voltage-indicator .sld-label {dominant-baseline: middle}
+.sld-voltage-indicator .sld-unpowered {stroke: none; fill: grey}
+.sld-voltage-indicator .sld-powered {stroke: none; fill: var(--sld-vl-color, black)}
```

Comment:
Ok, I got it, keeping only one component makes the things nicer here...
I think a way to go could be to keep one component only, but to add a new `BusVoltageInfo` class in the test which extends `BusInfo` and adds this annoying boolean. Then we need to add in the style provider interface a `Optional<String> getBusInfoStyle(BusInfo)` which you will override in the test, giving the style according to the boolean value.